app/assets/stylesheets/application.scss

```diff
@@ -11,4 +11,7 @@
 // Your CSS partials
 @import "components/index";
 @import "pages/index";
+@import "components/video_card";
```

Comment:
I think we can add all cards into one single component file _cards? Also, I think it's better to import all components into the _index.scss file which is then imported into application.scss instead of importing all components into appliations.scss separately.